pkg/kafka/auth/config_test.txt

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package auth
+
+import (
+	"testing"
+
+	"github.com/Shopify/sarama"
+	"github.com/spf13/viper"
+	//"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+
+	//"github.com/jaegertracing/jaeger/pkg/config/tlscfg"
+)
+
+
+
+func TestSetConfiguration(t *testing.T) {
+	logger, _ := zap.NewDevelopment()
+	saramaConfig := sarama.NewConfig()
+	v := viper.New()
+	configPrefix := "kafka.auth."
+	v.Set(configPrefix+suffixAuthentication, "kerberos")
```

Comment:
we don't test flags like that. See cmd/ingester/app/flags_test.go how we test flags.

Reply:
yes i have currently changed it but I am having some goleaks `TestSetConfiguration`